Patch: 02_force_autoconf_2.50.patch
Author: Lionel Elie Mamane <lionel@mamane.lu>
Get Debian's autoconf wrapper to use autoconf >= 2.50 (like upstream
uses) and not autoconf 2.13.
Index: b/configure.in
===================================================================
--- a/configure.in	2010-05-24 17:50:51.000000000 +0200
+++ b/configure.in	2010-05-24 17:50:59.000000000 +0200
@@ -16,7 +16,7 @@
 
 dnl Process this file with autoconf to produce a configure script.
 AC_REVISION($Revision: 8122 $)
-AC_PREREQ(2.0)
+AC_PREREQ(2.50)
 AC_INIT(src/common.h)
 
 
